docs/src/main/tut/index.md

```diff
@@ -8,31 +8,49 @@ permalink: /
 
 ## What is Mu?
 
-[Mu] provides the ability to combine [RPC] protocols, services, and clients in your Scala program, thanks to [gRPC]. 
-Although it's fully integrated with [gRPC], there are some important differences when defining the protocols, as we’ll see later on.
+[Mu] is a toolkit designed to make it easier to build and maintain
+microservices in a functional style.
+
+While you focus on implementing the business logic for your service, let Mu take
+care of the boilerplate, including:
+
+* generation of model classes, service interfaces and clients from [Avro],
+  [Protobuf] or [OpenAPI] <abbr title="Interface definition language">IDL</abbr> files
+* serialization of requests and responses into Avro/Protobuf/JSON
+* handling of streaming requests and responses using either [FS2] or [Monix]
+  Observables.
+* schema management and schema evolution
+* metrics reporting
+* ... and plenty more features on the way!
+
+Specifically, Mu helps you to build:
+
+* [gRPC] servers and clients based on either [Avro] or [Protobuf] protocol definitions
```

Comment:
```diff
-- gRPC
++ RPC
```

Reply:
Really? Isn't it reasonable to say `gRPC` here?

Reply:
afaicr Avro uses its own RPC

Reply:
In Mu, they both use gRPC. Here is the code to turn an avro4s codec into a gRPC marshaller: https://github.com/higherkindness/mu-scala/blob/81583cfb2787a6372e3e973ee8708e492e281fb7/modules/internal/src/main/scala/higherkindness/mu/rpc/internal/encoders/avro.scala#L49

Reply:
didn't know about this in mu, thanks :+1: